skore/src/skore/ui/project_routes.py

```diff
@@ -138,4 +146,27 @@ async def delete_view(request: Request, key: str):
             status_code=status.HTTP_404_NOT_FOUND, detail="View not found"
         ) from None
 
-    return __serialize_project(project)
+    return __project_as_serializable(project)
+
+
+@router.get("/activity")
+async def get_activity(
+    request: Request,
+    after: datetime = datetime(1, 1, 1, 0, 0, 0, 0, timezone.utc),
+):
+    """Send all recent as a JSON array.
+
+    The activity is composed of all the items and their versions created after the
+    datetime `after`.
+    """
+    project = request.app.state.project
+    return sorted(
+        (
+            __item_as_serializable(key, version)
+            for key in project.list_item_keys()
+            for version in project.get_item_versions(key)
+            if datetime.fromisoformat(version.updated_at) > after
+        ),
+        key=operator.attrgetter("updated_at"),
```

Comment:
I'm confused about the meaning of `updated_at` because of the version history. Can you remind me the difference between `updated_at` and `created_at`?

Reply:
`created_at` -> date of the first version on an item
`updated_at` -> date of the current version

Reply:
Thanks. As usual, I think eventually we should only have `created_at`, because Items themselves can't be modified; it's rather that the key that points to it may point to something else.

Reply:
You're write this debt that we need to reimburse soon.